examples/discovery-search-app/cypress/integration/basic/spec.ts

```diff
@@ -1,6 +1,6 @@
 describe('Basic search', () => {
   beforeEach(() => {
-    cy.server({ force404: true });
```

Comment:
Removed it, because it's getting set automatically in the `serverDefaults` file: https://github.com/watson-developer-cloud/discovery-components/blob/d9b633b51706604102d8c08541a96b56f2aac28d/examples/discovery-search-app/cypress/support/serverDefaults.ts#L2